features/specs/gezag-meerderjarige/gezag-over-kind-van-partner.feature

```diff
@@ -53,6 +53,7 @@ Functionaliteit: gezamenlijk gezag van een meerderjarige over een kind van een (
       | minderjarige.burgerservicenummer | 000000036        |
       | ouder.burgerservicenummer        | 000000024        |
       | derde.burgerservicenummer        | 000000012        |
+      | derde.type                       | Derde            |
```

Comment:
zie https://github.com/BRP-API/brp-api-gezag/issues/274#issuecomment-2633671768

bij voogdij wordt geen type toegevoegd, terwijl die volgens de OAS required is. Eén van de twee moet worden aangepast

Reply:
OAS is aangepast naar de laatste voorstellen in https://github.com/BRP-API/brp-shared/pull/19